compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -24,12 +22,13 @@ use crate::interpret::{
 };
 
 // Returns a pointer to where the result lives
+#[instrument(level = "debug", skip(ecx, body), ret)]
```

Comment:
```suggestion
#[instrument(level = "trace", skip(ecx, body), ret)]
```
pushing back against debug level inflation... the "trace" logs are already basically useless to me since I just drown in text when I enable that level, can we keep at least "debug" somewhat manageable?

Reply:
I just used `debug` as it was used in the body